src/System.Runtime.Extensions/src/System/IO/TextWriter.cs

```diff
@@ -109,7 +109,9 @@ public virtual string NewLine
         // but descendant classes can override the method to provide the
         // appropriate functionality.
         //
-        public abstract void Write(char value);
+        public virtual void Write(char value)
+        {
+        }
```

Comment:
I'm not arguing against changing it from abstract to virtual to match what it should be.  I'm just saying we should then take it a step further and make it actually do something.